cfgov/ask_cfpb/tests/test_search.py

```diff
@@ -304,33 +305,36 @@ def test_en_search_es7_no_term(self, mock_search):
         response = self.client.get(reverse("ask-search-en"), {"q": term})
         self.assertEqual(mock_search.call_count, 0)
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(response.context_data["page"], self.en_page)
-        self.assertEqual(response.context_data["page"].query, term)
-        self.assertEqual(response.context_data["page"].result_query, term)
+        response_page = response.context_data["page"]
+        self.assertEqual(response_page, self.en_page)
+        self.assertEqual(response_page.query, term)
+        self.assertEqual(response_page.result_query, term)
 
     @override_settings(FLAGS={"ASK_SEARCH_TYPOS": [("boolean", True)]})
     @override_settings(FLAGS={"ELASTICSEARCH_DSL_ASK": [("boolean", True)]})
     @mock.patch.object(AnswerPageDocument, 'search')
-    def test_en_search_suggestion_es7(self, mock_search):
+    def test_en_search_suggestion_es7(self, mock_suggestion):
```

Comment:
For readability, because what is being mocked is the `search()` method, this should probably stay named `mock_search`.
```suggestion
    def test_en_search_suggestion_es7(self, mock_search):
```

Reply:
Done